### ISSUE TITLE
Failed parsing for misconfigured parameters

### ISSUE THREAD

**Issue by Philius** [Motion-Project/motionplus#43]
### Did you read the guide?

Yes

### What is the base version number of MotionPlus being used?

0.1.x

### What was the install method?

Built from source code

### What is base architecture?

x86/x64

### What is the distro being used?

Debian

### Disto version number

12

### Camera/Sound type(s) being used?

v4l2 camera

### Describe the issue/problem and steps to reproduce

I'm testing with a USB camera with integrated microphone.
I tried:
`4l2_params "Focus, Automatic Continuous"=1,"Auto Exposure"=3,"White Balance, Automatic"=1`
and the log file became huge very quickly.
Substituting for the parameter IDs worked as expected:
`v4l2_params ID10094860=1,ID10094849=3,ID09963788=1`

### Relevant MotionPlus log output (at log_level 8)

```shell
Mar 20 17:49:53 [INF][ALL][00:ml00:MyCam1] mlp_setupmode: changes after 'EedDl':     0 - labels:   0 - noise level: 32
```

**Comment by Mr-Dave** [comment]
Please provide the first part of the log illustrating the parameters you specified as well as the up to the point that the setup messages appear.

**Comment by Mr-Dave** [comment]
You have specified setup mode.

**Comment by Philius** [comment]
The last two lines in the log file repeated a million times in the few seconds I ran the program.
(I snipped them off)
[motionplus.log](https://github.com/Motion-Project/motionplus/files/11039457/motionplus.log)
I still don't know what setup mode does, but I set it to off now.
This time around it was because set
`v4l2_params "Focus, Automatic Continuous"=1,"Auto Exposure"=3,"White Balance, Automatic"=1 palette=8`
which causes the big log file.
Setting
`v4l2_params "Focus, Automatic Continuous"=1,"Auto Exposure"=3,"White Balance, Automatic"=1,palette=8`
fixed it.
A space instead of a comma before `palette=8` was the cause.
